bin/nvh

```diff
@@ -515,6 +532,18 @@ function is_ok() {
   fi
 }
 
+#
+# Synopsis: can_use_xz
+#
+
+function can_use_xz() {
+  # nvh uses xz on Linux and macOS at this time, if the system supports it
+  if ! ([[ "$(uname -s)" = "Linux" ]] && [[ -n "$(command -v xz)" ]]) \
```

Comment:
`-n "$(command -v xz)"` tests whether there is a string. I would prefer test the command succeeds, or alternatively that the output exists, in case you get a warning message that command not found. Have a look at the other uses of `command -v`.

Reply:
Can do in a moment.

---

I was intrigued by your comment, and research instincts kicked in. I found this when investigating `command`:

<details>
  <summary>Info about <code>command</code> (click to expand)</summary>

```
$ man command
No manual entry for command
$ command -V command
command is a shell builtin
$ command --help
command: command [-pVv] command [arg ...]
    Execute a simple command or display information about commands.
    
    Runs COMMAND with ARGS suppressing  shell function lookup, or display
    information about the specified COMMANDs.  Can be used to invoke commands
    on disk when a function with the same name exists.
    
    Options:
      -p    use a default value for PATH that is guaranteed to find all of
            the standard utilities
      -v    print a description of COMMAND similar to the `type' builtin
      -V    print a more verbose description of each COMMAND
    
    Exit Status:
    Returns exit status of COMMAND, or failure if COMMAND is not found.
```

</details>
.

I tried to look up `command`, but it is virtually impossible to do a web search on, since all the relevant terms (command, bash, shell, builtin) are very generic.